.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4]
+        env: ['EXECUTOR= DEPENDENCIES=--prefer-lowest', 'EXECUTOR=coroutine DEPENDENCIES=--prefer-lowest', 'EXECUTOR=', 'EXECUTOR=coroutine']
+        include:
+          - os: ubuntu-latest
+            php: 7.1
+            env: EXECUTOR=coroutine
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install PHP
+      uses: shivammathur/setup-php@master
+      with:
+        php-version: ${{ matrix.php  }}
+        extension-csv: dom, mbstring
+        coverage: xdebug
+    - name: Show php version
+      run: php${{ matrix.php }} -v && composer -V
+    - name: Install dependencies
+      run: |
+        export $ENV
```

Comment:
What's in $ENV? Why is it needed?

Reply:
because env include more than one environment variable, export used for setting environment variables

Reply:
And isn't it already set? 🤔

Reply:
Pair of environment variables from level `jobs.build.strategy.matrix.env` will be mapped in env variable with name `ENV` inside step